catalog/opensearch/catalog-opensearch-source/src/main/java/ddf/catalog/source/opensearch/impl/OpenSearchParser.java

```diff
@@ -41,11 +41,9 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class OpenSearchParserImpl implements OpenSearchParser {
+public final class OpenSearchParser {
```

Comment:
What's the rationale behind making this final?  What if downstream projects wish to extend this class?

Reply:
I'm open to discussion about this, but my motivation was `OpenSearchParser` only contains some static methods to add some query parameters in a `WebClient`, and it should never be instantiated.

I would think that a new class
```java
public class AnotherProjectOpenSearchParser {

    public static void populateSomething(WebClient client,
```
and using the static methods from each class directly 
```java
AnotherProjectOpenSearchParser.populateSomething(webClient
```
```java
OpenSearchParser.populateSearchOptions(webClient
```
would work well.